src/aiperf/metrics/types/request_error_rate_metric.py

```
@@ -0,0 +1,37 @@
+    """
+
+    tag = "request_error_rate"
+    header = "Request Error Rate"
+    short_header = "Err %"
+    short_header_hide_unit = True
+    unit = GenericMetricUnit.PERCENT
+    display_order = 1080
+    flags = MetricFlags.NO_INDIVIDUAL_RECORDS
+    required_metrics = {
+        RequestCountMetric.tag,
+    }
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)